thoth/metrics_exporter/metrics.py

```diff
@@ -144,8 +156,14 @@
 )
 
 workflow_inspection_latency = Histogram(
-    "thoth_workflow_inspection_latency", "Thoth Inspection duration in Argo Workflow.", [],
-    buckets=[120, 240, 360, 480, 600]
+    "thoth_workflow_inspection_latency",
+    "Thoth Inspection duration in Argo Workflow.",
+    [],
+    buckets=[120, 240, 360, 480, 600],
```

Comment:
I don't want to nitpick, but will this work in prod deployment? I would expect "bucketing" to be done on the visualization layer as buckets can be skewed easily based on the actual load in the system (instead of having hardcoded buckets).

Reply:
If we don't set the bucket, Grafana will set default ones, which might be much lower. The idea was to start with some values and then adjust them accordingly depending on the SLO we want to achieve.